streams/src/test/java/org/apache/kafka/streams/processor/internals/RepartitionTopicsTest.java

```diff
@@ -151,22 +147,17 @@ public void shouldSetupRepartitionTopics() {
 
         assertThat(repartitionTopics.topologiesWithMissingInputTopics().isEmpty(), is(true));
         assertThat(repartitionTopics.missingSourceTopicExceptions().isEmpty(), is(true));
+
+        verify(copartitionedTopicsEnforcer).enforce(eq(coPartitionGroup1), any(), eq(clusterMetadata));
+        verify(copartitionedTopicsEnforcer).enforce(eq(coPartitionGroup2), any(), eq(clusterMetadata));
     }
 
     @Test
     public void shouldReturnMissingSourceTopics() {
         final Set<String> missingSourceTopics = mkSet(SOURCE_TOPIC_NAME1);
-        expect(internalTopologyBuilder.subtopologyToTopicsInfo())
-            .andReturn(mkMap(mkEntry(SUBTOPOLOGY_0, TOPICS_INFO1), mkEntry(SUBTOPOLOGY_1, TOPICS_INFO2)));
-        expect(internalTopologyBuilder.copartitionGroups()).andReturn(Collections.emptyList());
```

Comment:
Returning empty collections wherever it can is the default behaviour for Mockito, so a few of these have magically disappeared.